todo_app/.env.test

```diff
@@ -10,4 +10,8 @@ SECRET_KEY=secret-key
 USERNAME=TEST
```

Comment:
Not important for this module, but if you could update this to e.g. `DB_USER` instead, the `USERNAME` variable can cause issues on Windows machines (e.g. for me it always comes through with my windows account username, rather than the environment variable)

Reply:
I have been struggling to get the secrets changed on travis for this. Got some weird auth issues regenerating the encrypted version. Given that I develop in wsl and this is being run in linux docker containers I am not sure this is a big issue.